src/Tribe/Views/V2/Assets.php

```diff
@@ -419,7 +423,7 @@ public function register() {
 				'priority'     => 10,
 				'conditionals' => [ $this, 'should_enqueue_frontend' ],
 				'groups'       => [ static::$group_key, static::$widget_group_key ],
-				'in_footer'    => false,
+				'in_footer'    => array( 'strategy'  => 'defer' ),
```

Comment:
Minor code format tweak:

```suggestion
				'in_footer'    => [ 'strategy' => 'defer' ],
```